Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Barycentric_mapping_parameterizer_3.h

```diff
@@ -180,7 +182,8 @@ class Barycentric_mapping_parameterizer_3
       Vertex_around_target_circulator<Triangle_mesh> /* neighbor_vertex_v_j */ ) const
   {
     /// In the Tutte Barycentric Mapping algorithm, we have `w_ij = 1`, for `j` neighbor vertex of `i`.
-    return 1.;
+    const Point_3 stub;
+    return CGAL::Weights::uniform_weight(stub, stub, stub, stub);
```

Comment:
Is that at zero costs?

Reply:
Not zero but negligible. Anyway, you are right, I will keep `NT(1)` here. I tried to make the naming uniform with respect to the new weights that is why I use this function. But it is not really important in this case.